test/integration/che/che.go

```diff
@@ -26,13 +26,7 @@ import (
 )
 
 func applyingCheWithOpenshiftTokenSucceeds() error {
-	err := testsuite.MinishiftInstance.ExecutingOcCommand("whoami -t")
-	if err != nil {
-		return err
-	}
-
-	token := testsuite.GetLastCommandOutput().StdOut
-	err = testsuite.MinishiftInstance.ExecutingMinishiftCommand("addons apply --addon-env OPENSHIFT_TOKEN=" + token + " che")
+	err := testsuite.MinishiftInstance.ExecutingMinishiftCommand("addons apply che")
```

Comment:
If the token is no longer needed, then I suggest to remove this whole function + its Gherkin definition at https://github.com/minishift/minishift/blob/master/test/integration/che/che-steps.go#L38 and use the command directly in the feature file. Changing https://github.com/minishift/minishift/blob/master/test/integration/features/addons/addon-che.feature#L15 to `When executing "addons apply che" succeeds`.